tcp_check/test_tcp_check.py

```diff
@@ -15,18 +15,18 @@
     'instances': [{
         'host': '127.0.0.1',
         'port': 65530,
-        'timeout': 1,
+        'timeout': 1.5,
```

Comment:
Indeed this is not the first time we increase the timeouts, we've bumped both the instance timeouts, and the actual test timeout waiting for results. It's likely a timeout issue because behavior in `appveyor` is pretty consistent, but it does seem to be a bit inconsistent so this could bite again.